internal/commands/build.go

```diff
@@ -158,7 +167,16 @@ func doBuildCreate(ctx context.Context, hathora *sdk.SDK, appID *string, buildTa
 		zap.L().Error("failed to stream output to console", zap.Error(err))
 	}
 
-	return createRes.Build, nil
+	infoRes, err := hathora.BuildV2.GetBuildInfo(
```

Comment:
Could there be a brief enough lag between submitting and actually retrieving build information this could return an error?

Reply:
I suppose theoretically there could be, but it would return stale data rather than error, which I don't think is an issue

fwiw the current behavior in production is to _always_ return stale data and this fixes that